wordpress-mobile/WordPress-Android
# Remove classes and resources related to the old site creation flow

Fixes #9045 

Note!!
This PR removes `NewBlogActivity` which is being referenced from `create_wpcom_blog_from_settings.py`. AFAIK we don't use these scripts anymore, right? Should we remove them perhaps?

Removes everything related to the old site creation flow
- feature flag is removed in https://github.com/wordpress-mobile/WordPress-Android/commit/6c230926b78eedd954c80a71b9a23c61fb4ec614
- all classes and resources are removed in https://github.com/wordpress-mobile/WordPress-Android/commit/b824f195250ff9c01c6d34002e955c38439d6cc9
- zendesk "origin" string is updated in https://github.com/wordpress-mobile/WordPress-Android/commit/1a1cb398c504ea6107e7f6fc2b22e2fe7b7a4790

Make sure classes in `org.wordpress.android.ui.sitecreation` has not been changed (with exception of the zendesk "origin").

To test:
1. Walk through the site creation flow and make sure the app works as expected - also try to click on the "HELP" button in the tooolbar on each step of the flow.

Update release notes:

- There are no user facing changes
